frontend/src/pages/Incidences/components/IncidenceForm.tsx

```
@@ -9,27 +9,34 @@ import { Checkbox } from "../../../components/ui/checkbox"
 import { IncidenceService, type LocationType } from "../../../services/incidences"
+import { useStaff } from "../../Staff/hooks/useStaff"
 
 interface IncidenceFormProps {
   onSuccess: () => void
   onClose: () => void
+  isAdmin?: boolean
 }
 
-export function IncidenceForm({ onSuccess, onClose }: IncidenceFormProps) {
+export function IncidenceForm({ onSuccess, onClose, isAdmin = false }: IncidenceFormProps) {
+  const { staff = [], loading: loadingStaff } = isAdmin ? useStaff() : { staff: [], loading: false };
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)